hygieia-jenkins-plugin/README.md

```diff
@@ -2,8 +2,7 @@
 
 # Developer instructions
 
-Install Maven and JDK.  This was last built with Maven 3.2.5 and OpenJDK
-1.7.0\_75 
+Install Maven and JDK.  This was last built with Maven 3.2.5 and JDK 1.8. 
```

Comment:
@tabladrum can you see if you can build this with maven 3.3.9 . The rest of the project was built with that version of maven